config/rbac/aggregate_to_view.yaml

```diff
@@ -0,0 +1,16 @@
+# Automatically aggregated to clusterRoles: cluster-reader, view -> edit -> admin
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+  name: hnc-view
```

Comment:
I don't *think* you need the `hnc-` prefix here, shouldn't Kustomize add that for you?

Reply:
Maybe yes, I didnt saw prefix configuration in local kustomization file, so idk.  
I will remove it tho